pkgs/os-specific/linux/systemd/default.nix

```diff
@@ -591,7 +581,7 @@ stdenv.mkDerivation {
         ''
           if [[ $(grep -r '${search}' | grep -v "${replacement}" | grep -Ev 'NEWS|^test/' | wc -l) -gt 0 ]]; then
             echo "Not all references to '${search}' have been replaced. Found the following matches:"
-            grep '${search}' -r | grep -v "${replacement}" | grep -Ev 'NEWS|^test/'
+            grep '${search}' -r | grep -v "${replacement}" | grep -Ev 'NEWS|^test/' | grep -Ev "${}"
```

Comment:
I guess you meant to add `ignore` support here. I think we shouldn't ignore these files, because `tar` strings in them refer to `systemd-pull tar` argument, which is already replaced with Nix store path in `src/import/pull-tar.c` and others.

Overall `systemd-pull` seems to be broken currently and I think we should fix this in a separate PR.